src/development/platform-integration/platform-channels.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: Writing custom platform-specific code with platform channels
+title: Writing custom platform-specific code
+short-title: Platform-specific code
```

Comment:
I've updated the title to match what was used in sidenav.yml. Is this ok @Sfshaza?

Reply:
Yes. It was SUCH a long name...